inc/Widget/SmailyWidget.php

```diff
@@ -107,10 +107,67 @@ public function widget( $args, $instance ) {
 				<div class="smaily-newsletter-alert">
 				<p>' . esc_html( $message ) . '
 				<span class="smaily-newsletter-closebtn" onclick="this.parentElement.style.display=\'none\'">&times;</span>
-				</p>
+					<p>' . esc_html( $message ) . '
+						<span class="smaily-newsletter-closebtn" onclick="this.parentElement.style.display=\'none\'">&times;</span>
+					</p>
 				</div>
 				';
 		}
+
+		// Layout options to echo.
+		if ( isset( $instance['form_layout'] ) ) {
+			switch ( $instance['form_layout'] ) {
+				// Layout 1.
+				case 'layout-1':
+					$layout = '<div class="smaily-layout-1-container">
+						<input class="smaily-layout-1-input" type="text" name="email" value="" placeholder="' . esc_attr( $instance['email_field_placeholder'] ) . '" />
+						<button class="ui pink basic button smaily-button" type="submit" style="background-color: ' . esc_attr( $instance['submit_button_color'] ) . ' ; color: ' . esc_attr( $instance['submit_button_text_color'] ) . '" >' . esc_attr( $instance['submit_button_text'] ) . '</button>
+					</div>';
+					break;
+				// Layout 2.
+				case 'layout-2':
+					$layout = '<div class="smaily-layout-2-container">
+						<input type="text" name="email" value="" placeholder="' . esc_attr( $instance['email_field_placeholder'] ) . '" />
```

Comment:
With some themes, `layout-2` text field can have spacing between text field and button. Please make sure every margin between those two are removed.